test/parallel/test-crypto-key-objects.js

```diff
@@ -156,7 +158,16 @@ const privateDsa = fixtures.readKey('dsa_private_encrypted_1025.pem',
   assert(Buffer.isBuffer(privateDER));
 
   const plaintext = Buffer.from('Hello world', 'utf8');
-  const ciphertexts = [
+  const testDecryption = (fn, ciphertexts, decryptionKeys) => {
+    for (const ciphertext of ciphertexts) {
+      for (const key of decryptionKeys) {
+        const deciphered = fn(key, ciphertext);
+        assert(plaintext.equals(deciphered));
```

Comment:
```suggestion
        assert.strictEqual(deciphered, plaintext);
```

Reply:
I believe that won't work since `plaintext` and `deciphered` are buffers. I think `deepStrictEqual` would work if you prefer that.

Reply:
Oh, you're right. But yes, I think that'd be better.